packages/app/src/app/pages/SignIn/Onboarding.tsx

```diff
@@ -114,14 +121,23 @@ export const Onboarding = () => {
             }}
             value={newUsername}
             onChange={e => setNewUsername(e.target.value)}
+            required
+            isInvalid={pendingUser.valid === false}
```

Comment:
_nit_: (should've been caught by prettier)

```suggestion
            isInvalid={pendingUser.valid === false}
            required
```